server/src/services/app_import_export.service.ts

```diff
@@ -57,6 +57,7 @@ const DefaultDataSourceNames: DefaultDataSourceName[] = [
   'workflowsdefault',
 ];
 const DefaultDataSourceKinds: DefaultDataSourceKind[] = ['restapi', 'runjs', 'runpy', 'tooljetdb', 'workflows'];
+const NewRevampedComponents = ['Text', 'TextInput', 'PasswordInput', 'NumberInput', 'DropDown'];
```

Comment:
Using an enum for `NewRevampedComponents` improves code safety, readability, and maintenance